pkg/capabilities/mercury/mercury.go

```diff
@@ -51,6 +52,21 @@ type ReportInfo struct {
 	Price     float64
 }
 
+// TODO: fix this by adding support for uint64 in value.go
+type FeedReport struct {
+	FeedID               int64  `json:"feedId"`
+	Fullreport           []byte `json:"fullreport"`
+	BenchmarkPrice       int64  `json:"benchmarkPrice"`
+	ObservationTimestamp int64  `json:"observationTimestamp"`
```

Comment:
We probably want the expiresAt and validFrom fields, the observation timestamp by itself isnt very useful to us

Reply:
NVM, this might be wrong. Confirming with austin first. Ignore for now.